app/src/main/java/org/mozilla/fenix/home/recenttabs/RecentTabsListFeature.kt

```diff
@@ -28,20 +28,42 @@ class RecentTabsListFeature(
 ) : AbstractBinding<BrowserState>(browserStore) {
 
     override suspend fun onState(flow: Flow<BrowserState>) {
+        lateinit var recentTabs: ArrayList<RecentTab>
```

Comment:
Why do we need to hold a reference here?

Reply:
I think I know what you're trying to do...

```kotlin
flow.map { it.asRecentTabs() }
    // Listen for changes regarding the currently selected tab and the in progress media tab
    // and also for changes (close, undo) in normal tabs that could involve these.
    .ifChanged()
    .collect {
        homeStore.dispatch(HomeFragmentAction.RecentTabsChange(it))
    }
```

Reply:
Yes! this is what I've been looking for.  Thanks!